data/gnome-shell-theme.gresource.xml

```diff
@@ -48,5 +48,27 @@
     <file>hot-corner-rtl-symbolic.svg</file>
     <file>mini-icon-active-indicator.png</file>
     <file>system-logout.png</file>
+
```

Comment:
Actually, this commit shouldn't have made it to this PR, it's part of my WIP branch. I'll remove it